specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2021-01-01/attestations.json

```diff
@@ -639,6 +639,9 @@
           "items": {
             "$ref": "#/definitions/AttestationEvidence"
           },
+          "x-ms-identifiers": [
+            "sourceUri"
```

Comment:
>sourceUri [](http://example.com/codeflow?start=13&length=9)

Items in this array are not guaranteed to be unique.  It could contain the same evidence 10 times if they user wants to do that